README.md

```diff
@@ -6,5 +6,36 @@ are not redirected but evaluated in the bridge as they require the JVM.
 [nREPL]: https://github.com/clojure/tools.nrepl
 [Arcadia Unity]: https://github.com/arcadia-unity/Arcadia
 
-## How to use
-TBD
+### How to use Cursive with Arcadia
```

Comment:
This section assumes we already have Clojure CLI, a Unity project, and Arcadia, so perhaps we can add a Prerequisites section with more details;
- Link to [Clojure Guide](https://clojure.org/guides/getting_started)
- Link to how to setup [Arcadia with Unity](https://github.com/arcadia-unity/Arcadia/wiki/Getting-Started)